templates/kops-instancegroup-master.tpl.yaml

```diff
@@ -47,4 +47,19 @@ ${subnets}
       OnUnitActiveSec=60min
 
       [Install]
-      WantedBy=timers.target
\ No newline at end of file
+      WantedBy=timers.target
+  - name: calico-fix.service
+    manifest: |
+      [Unit]
+      Description=Calico network fix
+
+      [Service]
+      Type=oneshot
+      ExecStart=-/bin/bash -c "mv /etc/cni/net.d/10-calico.conflist /etc/cni/net.d/10-calico.conf"
+      ExecStartPost=-/bin/systemctl restart kubelet.service
```

Comment:
Not needed. The kubelet scans repetitively, so restarting it is not needed. I tried this manually only renaming the file and it worked.